chartfx-chart/src/main/java/de/gsi/chart/renderer/spi/ContourDataSetCache.java

```diff
@@ -81,11 +82,13 @@
     protected double[] tempDataBuffer;
     protected final double[] reduced;
 
-    public ContourDataSetCache(final XYChart chart, final ContourDataSetRenderer renderer, final DataSet dataSet) {
+    public ContourDataSetCache(final XYChart chart, final ContourDataSetRenderer renderer, final GridDataSet dataSet) {
```

Comment:
Codacy found an issue: [The constructor 'ContourDataSetCache(XYChart, ContourDataSetRenderer, GridDataSet)' has an NPath complexity of 256](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)